From: Jonas Meurer <jonas@freesources.org>
Date: Fri, 15 Dec 2017 00:39:13 +0100
Description: example_project: Fix path to urls for tests
 Fix the path to urls to run the postorius test suite.
Forwarded: not-needed

---
 example_project/settings.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/example_project/settings.py b/example_project/settings.py
index 6220de4..85f37bd 100644
--- a/example_project/settings.py
+++ b/example_project/settings.py
@@ -97,7 +97,7 @@ MIDDLEWARE_CLASSES = (
 
 # Set `postorius.urls` as main url config if Postorius
 # is the only app you want to serve.
-ROOT_URLCONF = 'urls'
+ROOT_URLCONF = 'example_project.urls'
 
 
 TEMPLATES = [
